scripts/release.sh

```diff
@@ -2,288 +2,154 @@
 
 set -e
 
-# shellcheck source=scripts/lib/image_defs
-. "${DAPPER_SOURCE}/scripts/lib/image_defs"
-# shellcheck source=scripts/lib/utils
-. "${DAPPER_SOURCE}/scripts/lib/utils"
-. "${SCRIPTS_DIR}/lib/debug_functions"
+source ${DAPPER_SOURCE}/scripts/lib/image_defs
+source ${DAPPER_SOURCE}/scripts/lib/utils
+source ${SCRIPTS_DIR}/lib/debug_functions
```

Comment:
Did commits get mixed up here?

Reply:
If you go commit-by-commit you'll see the very first one renames `release.sh` to `do-release.sh` and that gets picked up by GitHub correctly:

![image](https://user-images.githubusercontent.com/51078027/123223688-c7340900-d4d9-11eb-9851-8b65aff8fac6.png)

Unfortunately, in the "unified PR view" it looks as if I'm rewriting the whole file, and makes it confusing.

I suggest to follow commit-by-commit which should be clearer.

Reply:
Yes, I reviewed the code locally which made it easier to understand. What misled me initially was the difference in style at the start of `do-release.sh` and `release.sh`!